test/gpu/native/errors/std/MyMod.chpl

```diff
@@ -0,0 +1,30 @@
+module MyMod {
+
+  config param usePoi = true;
+
+  pragma "no gpu codegen"
+  proc doSomethingElse(x) {}
+
+  proc doSomething(x) {
+    doSomethingElse(x);
+  }
+
+  proc doSomethingElsePOI(x) {
```

Comment:
Out of curiosity: what does POI stand for.

It looks like the `doSomething` path always fails (inside the module code) and the `doSomethingPOI` calls out to `convertToString` (presumably implemented in user code) which may or may not fail depending on the particulars of how `convertToString` is implemented for some type `x`.

Reply:
It stands for [_point of instantiation_](https://chapel-lang.org/docs/language/spec/generics.html#function-visibility-in-generic-functions). Specifically, generic functions can also search the scope in which they're instantiated for symbols. Thus, even though the "standard" module in the test doesn't know what `toString` is (and it's not in scope), it can still be made to call `toString` because it searches the test file's scope (its instantiation point).

POI is the only way I can think of to get standard library code to execute user code again.